plugins/ufm_log_analyzer_plugin/unit_tests/test_ibdiagnet_log_analyzer.py

```diff
@@ -0,0 +1,46 @@
+"""
+@copyright:
+    Copyright © 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+
+    This software product is a proprietary product of Nvidia Corporation and its affiliates
+    (the "Company") and all right, title, and interest in and to the
+    software product, including all associated intellectual property rights,
+    are and shall remain exclusively with the Company.
+
+    This software product is governed by the End User License Agreement
+    provided with the software product.
+
+@author: Miryam Schwartz
+@date:   Dec 08, 2024
+"""
```

Comment:
From what I know the copyright is usually a comment and not a string

Reply:
changed